src/main/java/com/questhelper/steps/DetailedQuestStep.java

```diff
@@ -122,10 +125,12 @@ public class DetailedQuestStep extends QuestStep
 	public boolean considerBankForItemHighlight;
 	public int iconToUseForNeededItems = -1;
 
+
 	public DetailedQuestStep(QuestHelper questHelper, String text, Requirement... requirements)
 	{
 		super(questHelper, text);
 		this.requirements.addAll(Arrays.asList(requirements));
+
```

Comment:
Not sure why you're adding new lines at the end of methods, is this part of the formatting rules? Doesn't look like it on first sight https://github.com/runelite/runelite/wiki/Code-Conventions